### PR TITLE
Issue 2 show categories

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-09-13 at 5 20 43 PM
Screen Shot 2022-09-13 at 5 21 14 PM

https://user-images.githubusercontent.com/106217355/190031835-7988a254-fc03-4ac1-832b-99357abd3cb9.mov

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -1,14 +1,68 @@
 import React from 'react';
-import Home from './pages/home';
+// import Home from './pages/home';
```
Comment (thebearingedge) [inline client/app.jsx:2]: Please remove commented code.

File: client/app.jsx [hunk 2]
```diff
@@ -1,14 +1,68 @@
...
   render() {
     return (
       <>
-        <Navbar />
-        <Home />;
+        <Navbar onClick={this.getList}/>
+        <div className='container'>
+          <BookList books={this.state.books} />
+        </div>
+        {/* <Home />; */}
```
Comment (thebearingedge) [inline client/app.jsx:63]: Please remove commented code.

File: package.json [hunk 3]
```diff
@@ -13,11 +13,13 @@
   },
   "dependencies": {
     "argon2": "^0.29.1",
+    "bootstrap": "^5.2.1",
     "dotenv": "^16.0.2",
     "express": "^4.18.1",
     "jsonwebtoken": "^8.5.1",
     "pg": "^8.8.0",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
```
Comment (thebearingedge) [inline package.json:22]: if you are not using them you can uninstall them

```shell
npm uninstall bootstrap react-bootstrap
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by yuliia-p** [comment]
Screen Shot 2022-09-13 at 5 35 01 PM

File: package.json (hunk 3 above)
Comment (yuliia-p) [reply package.json]: Screen Shot 2022-09-13 at 5 35 28 PM

File: client/app.jsx (hunk 2 above)
Comment (yuliia-p) [reply client/app.jsx]: Screen Shot 2022-09-13 at 5 35 52 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**